matRad/dicom/@matRad_DicomImporter/matRad_importDicomCt.m

```diff
@@ -65,7 +65,7 @@
     % might not been defined for individual files
     obj.importCT.ctInfo(i).PixelSpacing            = tmpDicomInfo.PixelSpacing;
     obj.importCT.ctInfo(i).ImagePositionPatient    = tmpDicomInfo.ImagePositionPatient;
-    obj.importCT.ctInfo(i).SliceThickness          = obj.importFiles.resz;
+    obj.importCT.ctInfo(i).SliceThickness          = str2double(obj.importFiles.resz);
```

Comment:
Are you certain that this is always a string / char array? Because if it is a number, it will become NaN here.